tests/test_ble_interface_core.py

```
@@ -411,9 +411,178 @@ def test_ble_interface_defaults_auto_reconnect_disabled(
+        iface,
+        "findDevice",
+        lambda _address: _create_ble_device("AA:BB:CC:DD:EE:FF", "Meshtastic"),
+    )
+
+    temp_client = SimpleNamespace(
+        pair=lambda **_kwargs: "paired",
+        bleak_client=SimpleNamespace(address="AA:BB:CC:DD:EE:FF"),
+    )
+    cleanup_calls: list[Any] = []
+
+    def _temp_client_factory(_address: str, **_kwargs: Any) -> Any:
```

Ruff (0.15.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**_kwargs`

(ANN401)